app/assets/javascripts/plugins/thermometers.js

```diff
@@ -21,7 +21,7 @@ $(function(){
 
   if($bar.length > 0) {
     getCount();
-    setInterval(getCount, 3000);
+    setInterval(getCount, 90000);
```

Comment:
I think it's fine either way, a bit overkill as it stands, but I reckon we can just leave it for now and either put it to work or axe it when we start conforming to the designs.